homeassistant/components/device_tracker/google_maps.py

```diff
@@ -12,12 +12,13 @@
 from homeassistant.components.device_tracker import (
     PLATFORM_SCHEMA, SOURCE_TYPE_GPS)
 from homeassistant.const import ATTR_ID, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.const import ATTR_BATTERY_CHARGING, ATTR_BATTERY_LEVEL
```

Comment:
Don't split imports from a single module into two statements.

Reply:
Fixed it.